.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "assignment-autotest"]
 	path = assignment-autotest
 	url = https://github.com/cu-ecen-aeld/assignment-autotest.git
+[submodule "test/assignment-autotest"]
+	path = test/assignment-autotest
+	url = https://github.com/cu-ecen-aeld/assignment-autotest.git
```

Comment:
@erpe9416 not sure where this change came from but I don't think you need or want this additional submodule.